edb/pgsql/compiler/pathctx.py

```diff
@@ -959,6 +959,10 @@ def get_path_serialized_output(
     # must be kept outside of get_path_output() generic.
     aspect = 'serialized'
 
+    view_path_id_map = getattr(rel, 'view_path_id_map', None)
```

Comment:
Untyped?

Reply:
Hm. It's a copypasta from a similar method. Might actually not be needed anymore.